tests/python/pants_test/backend/jvm/tasks/jvm_compile/rsc/test_rsc_compile_integration.py

```diff
@@ -40,18 +52,22 @@ def test_basic_binary(self):
       self.assert_is_file(rsc_header_jar)
 
   @ensure_compile_rsc_execution_strategy
-  def test_executing_multi_target_binary(self):
+  def executing_multi_target_binary(self):
     pants_run = self.do_command('run', 'examples/src/scala/org/pantsbuild/example/hello/exe')
     self.assertIn('Hello, Resource World!', pants_run.stdout_data)
 
   @ensure_compile_rsc_execution_strategy
-  def test_java_with_transitive_exported_scala_dep(self):
+  def java_with_transitive_exported_scala_dep(self):
     self.do_command('compile', 'testprojects/src/scala/org/pantsbuild/testproject/javadepsonscalatransitive:java-in-different-package')
 
   @ensure_compile_rsc_execution_strategy
-  def test_java_sources(self):
+  def java_sources(self):
     self.do_command('compile', 'testprojects/src/scala/org/pantsbuild/testproject/javasources')
 
+  @ensure_compile_rsc_execution_strategy
+  def node_dependencies(self):
+    self.do_command('compile', 'contrib/node/examples/src/java/org/pantsbuild/testproject/jsresources')
```

Comment:
Thanks for finding the target!

This would probably require adding `contrib/node:examples_directory` as a dep. E.g https://github.com/pantsbuild/pants/blob/master/tests/python/pants_test/backend/jvm/tasks/jvm_compile/rsc/BUILD#L11